lib/seed/facility_seeder.rb

```diff
@@ -64,7 +64,7 @@ def call
       facility_groups = number_of_facility_groups.times.map {
         FactoryBot.build(:facility_group, organization_id: organization.id, state: nil)
       }
-      fg_result = FacilityGroup.import(facility_groups, returning: [:id, :name], on_duplicate_key_ignore: true)
+      fg_result = FacilityGroup.import_with_regions(facility_groups, returning: [:id, :name], on_duplicate_key_ignore: true)
```

Comment:
We won't need this when https://github.com/simpledotorg/simple-server/pull/1571 lands.

Reply:
We can ditch this since you've inlined stuff, which looks good, but I was thinking I'd keep this anyway since it'd be nice to replace this

```
import_facilities.each(&:make_region) if Flipper.enabled?(:regions_prep) 
```

with

```
Facility.import_with_regions(import_facilities, validate: true)
```

in application code under `app/jobs/import_facilities_job.rb`.